invalid_coupon_code.py

```diff
@@ -1,3 +1,13 @@
+'''
+This script will input an invalid coupon code once an item has been checked out from the cart.

```

Comment:
the comment is not correct. 'has been checked out from the cart'. It is 'after user adds item to cart'.